test/Microsoft.DotNet.Cli.Utils.Tests/project.json

```diff
@@ -1,7 +1,8 @@
 {
   "version": "1.0.0-*",
   "compilationOptions": {
-    "emitEntryPoint": true
+    "emitEntryPoint": true,
+    "keyFile": "../../tools/Key.snk"
```

Comment:
In order for InternalsVisibleTo to work.  The product assembly is strong named.  In order for the product to have InternalsVisibleTo a different assembly, that assembly must be strong named as well.